crates/viewer/re_view_time_series/src/line_visualizer_system.rs

```diff
@@ -41,15 +40,21 @@ const DEFAULT_STROKE_WIDTH: f32 = 0.75;
 
 impl VisualizerSystem for SeriesLineSystem {
     fn visualizer_query_info(&self) -> VisualizerQueryInfo {
-        let mut query_info = VisualizerQueryInfo::from_archetype::<archetypes::Scalar>();
+        let mut query_info = VisualizerQueryInfo::from_archetype::<archetypes::Scalars>();
         query_info.queried.extend(
-            SeriesLine::all_components()
+            archetypes::SeriesLines::all_components()
                 .iter()
                 .map(|descr| descr.component_name),
         );
 
-        query_info.indicators =
-            std::iter::once(SeriesLine::descriptor_indicator().component_name).collect();
+        query_info.indicators = [
+            // Support deprecated `SeriesLine` archetype.
+            #[allow(deprecated)]
+            archetypes::SeriesLine::descriptor_indicator().component_name,
+            archetypes::SeriesLines::descriptor_indicator().component_name,
+        ]
+        .into_iter()
+        .collect();
```

Comment:
I would love to handle this as a migration during ingestion instead, i.e. translate the old deprecated archetypes/components to the new non-deprecated ones